### ISSUE TITLE
Releases and arm64-v8a support

### ISSUE THREAD

**Issue by k0bura** [jmriego/TaskwarriorC2#2]
I'm having struggles setting up my android studio to compile this as is with your latest code. Mainly conflicts with Lima1 dependencies. As an example:

`Duplicate class android.support.v4.app.ActionBarDrawerToggle found in modules support-core-ui-27.0.1-runtime (com.android.support:support-core-ui:27.0.1) and support-v4-23.2.1-runtime (com.android.support:support-v4:23.2.1)`.

It was a rabbit hole updating all Android and Node dependecies to get this to compile on my Mac M1 (the electron version is too old to run). 

Since you already have it running, can you post the binaries (apk) into the releases section? I'm also requesting arm64-v8a support because my phone only supports 64bit apps.

**Comment by jmriego** [comment]
yes, I'm having issues as well. I changed to a new phone (which is not arm64) and I'm completely unable to compile this in the newest Android Studio release. I haven't been able to fix it so I just stopped using TaskWarrior.
I was able to do an `adb pull` to get it from my old phone so I hope the release works for you.

Good luck!
